diff -up kmousetool-19.08.2/kmousetool/kmousetool.cpp.omv~ kmousetool-19.08.2/kmousetool/kmousetool.cpp
--- kmousetool-19.08.2/kmousetool/kmousetool.cpp.omv~	2019-10-15 14:22:37.261627224 +0200
+++ kmousetool-19.08.2/kmousetool/kmousetool.cpp	2019-10-15 14:22:44.081656205 +0200
@@ -30,6 +30,7 @@
 #include <X11/StringDefs.h>    /* Standard Name-String definitions*/
 #include <X11/extensions/xtestext1.h>    /* Standard Name-String definitions*/
 #include <X11/extensions/XTest.h>    /* Standard Name-String definitions*/
+#undef None
 
 #include <QAbstractEventDispatcher>
 #include <QApplication>
